client/my-sites/stats/stats-tabs/index.jsx

```diff
@@ -41,6 +51,7 @@ class StatsTabs extends Component {
 					tabClick: switchTab,
 					value: hasData ? activeData[ tab.attr ] : null,
 					format: tab.format,
+					iconSize: iconSize,
```

Comment:
```suggestion
					iconSize,
```

Extremely minor nit 🙂